### PR TITLE
AVT-990 [Spectron][Windows] Add test "Verify Always on Top options when multiple applications are opened"

### PR DESCRIPTION
**Pull request by thaisym1912**
## Description
Add test "Verify Always on Top options when multiple applications are opened" [AVT-990](https://perzoinc.atlassian.net/browse/AVT-9990)

## Run
Please refer attachment for more details
[Jest Test Report.zip](https://github.com/symphonyoss/SymphonyElectron/files/2204279/Jest.Test.Report.zip)

### CODE REVIEW

File: tests/spectron/alwaysOnTop.spectron.js [hunk 1]
```diff
@@ -1,190 +1,65 @@
...
-        }
+    /**
+     * Verify Always on Top options when multiple applications are opened
+     * TC-ID: 2898431
+     * Cover scenarios in AVT-990
+     */
+    it('Verify Always on Top options when multiple applications are opened', async (done) => {
+        try {
+            await webActions.minimizeWindows();
+            await windowActions.openAppInMaximize("C:\\Windows\\notepad.exe");
```
Comment (vantruongt2) [inline tests/spectron/alwaysOnTop.spectron.js]: Should move `openAppInMaximize` method to other.

File: tests/spectron/alwaysOnTop.spectron.js [hunk 2]
```diff
@@ -1,190 +1,65 @@
...
+            await windowActions.openAppInMaximize("C:\\Windows\\system32\\mspaint.exe");
```
Comment (vantruongt2) [inline tests/spectron/alwaysOnTop.spectron.js]: Same here

File: tests/spectron/spectronSetup.js [hunk 3]
```diff
@@ -26,22 +27,36 @@ class App {
...
-    startApplication(configurations) {
-        return this.app.start().then((app) => {
-            if (configurations)
-            {
-                if (configurations.alwaysOnTop)  {
-                    app.browserWindow.setAlwaysOnTop(true);
+    async startApplication(configurations) {
+        try {
+            this.app = await this.app.start();
+            await this.app.client.waitForVisible(ui.TITLE_BAR, 60000);
```
Comment (vantruongt2) [inline tests/spectron/spectronSetup.js]: Should define the variable for TIME_OUT

File: tests/spectron/spectronWindowsActions.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            robot.dragMouse(x, y);
+            robot.mouseToggle("up");
+        })
+    }
+
+    async getCurrentPosition() {
+        return this.app.browserWindow.getPosition();
+    }
+
+    async setPosition(x, y) {
```
Comment (vantruongt2) [inline tests/spectron/spectronWindowsActions.js]: Remove x,y argument if you don't use them

Comment (thaisym1912) [reply tests/spectron/spectronWindowsActions.js]: Updated

File: tests/spectron/spectronSetup.js (hunk 3 above)
Comment (thaisym1912) [reply tests/spectron/spectronSetup.js]: Updated

File: tests/spectron/alwaysOnTop.spectron.js (hunk 2 above)
Comment (thaisym1912) [reply tests/spectron/alwaysOnTop.spectron.js]: Updated

File: tests/spectron/alwaysOnTop.spectron.js (hunk 1 above)
Comment (thaisym1912) [reply tests/spectron/alwaysOnTop.spectron.js]: Updated

**Comment by VishwasShashidhar** [comment]
@vantruongt2 Please rebase this PR.

**Comment by thaisym1912** [comment]
@VishwasShashidhar Completed rebase

**Comment by thaisym1912** [comment]
Updated the test with Menu selection and added results
[Jest Test Report.zip](https://github.com/symphonyoss/SymphonyElectron/files/2218222/Jest.Test.Report.zip)

File: package.json [hunk 5]
```diff
@@ -109,6 +112,7 @@
   },
   "dependencies": {
     "@paulcbetts/system-idle-time": "1.0.4",
+    "applescript": "^1.0.0",
```
Comment (KiranNiranjan) [inline package.json]: This should be a devDependency

File: tests/spectron/spectronUtils.js [hunk 6]
```diff
@@ -1,8 +1,26 @@
 const childProcess = require('child_process');
+const { isMac } = require('../../js/utils/misc.js');
+// const applescript = require('applescript');
```
Comment (KiranNiranjan) [inline tests/spectron/spectronUtils.js]: Remove commented out code

File: tests/spectron/spectronUtils.js [hunk 7]
```diff
@@ -1,8 +1,26 @@
...
+                if (err) {
+                    console.log(err);
+                }
+                if (Array.isArray(rtn)) {
+                    rtn.forEach(function (songName) {
+                        console.log(songName);
+                    });
+                }
+            });
+            //await applescript.execString('tell application "TextEdit" tell window 1 set zoomed to true end tell end tell');
```
Comment (KiranNiranjan) [inline tests/spectron/spectronUtils.js]: same here

**Pull request closed without merging**